src/module-server/bundle-npm-module.ts

```diff
@@ -70,10 +70,9 @@ export const bundleNpmModule = async (
           },
           load(id) {
             if (id === virtualEntry) {
-              const code = `export * from '${mod}'
-export {${namedExports.join(', ')}} from '${mod}'
-export { default } from '${mod}'`;
-              return code;
+              return `export * from ${JSON.stringify(mod)}
+              export {${namedExports.join(', ')}} from ${JSON.stringify(mod)}
+              export { default } from ${JSON.stringify(mod)}`;
```

Comment:
Does it make sense to stringify it once?

```suggestion
              const normalizedMod = JSON.stringify(mod);
              return `export * from ${normalizedMod}
              export {${namedExports.join(', ')}} from ${normalizedMod}
              export { default } from ${normalizedMod}`;
```

(Not sure about var name.)